modules/machine-health-checks-baremetal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * machine_management/deploying-machine-health-checks.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-health-checks-baremetal_{context}"]
```

Comment:
You don't need to add another module. You need to weave this content into the existing modules.